# QuiOXAte overview

![](assets/quiOXAte.png)

- Wildly overambitious/quixotic attempt to "fix" AMR nomenclature 

- **Ultimate goal**: 
    - A workflow to generate a dynamically updated genomically informed AMR nomenclature to facilitate bioinformatic analyses.
    - By providing a common nomenclature onto which existing AMR genes can be mapped we can create a "backend" that retains existing AMR names and enables cross-database annotation mining/comparison.

- **Proximate goal**:
    - Focus on one of the worse AMR "gene families" the OXA beta-lactamases (many "OXA-XX" not actually related to one another and with diverse phenotypes) and capture the sampled diversity.
    - Deduplicate and generate a phylogeny then map drug annotations onto this tree
    - Annotate this tree with drug class information
    - Use relative evolutionary divergence measures to cut this tree into "equally diverse" (functionally/genically) hierarchical groupings e.g., family, subfamily, allele 

